packages/core/src/scd-plugin/stand-alone-mode.scd-plugin.ts

```diff
@@ -1,11 +1,20 @@
 // UTILS
 import { serializeXmlDocument } from '../scd-xml'
+// OPENSCD
+import {
+	isMove,
+	isCreate,
+	isDelete,
+	isReplace,
+	isUpdate
+} from '../scd-events-v1'
 // TYPES
 import type {
 	PluginConstructor,
 	EditorPluginInstance,
 	PluginType
 } from './types.scd-plugin'
+import type { Create, Delete, Move, Replace, Update } from '../scd-events-v1'
 
 export default function standAloneMode(
```

Comment:
`emulateOpenSCDInstanceEditionActions` can be separated into event listeners and handlers

Reply:
Refactored see 9fded53